packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -456,6 +470,11 @@ class _ExpansionPanelListState extends State<ExpansionPanelList> {
 
   @override
   Widget build(BuildContext context) {
+    assert(kElevationToShadow.containsKey(widget.elevation) != false,
```

Comment:
nit: already returns a bool, so no need to use an equality here:
```suggestion
    assert(!kElevationToShadow.containsKey(widget.elevation),
```